Cargo.toml

```diff
@@ -18,9 +18,10 @@ appveyor = { repository = "amethyst/amethyst" }
 travis-ci = { repository = "amethyst/amethyst" }
 
 [features]
-default = ["opengl"]
+default = ["opengl", "profiler"]
```

Comment:
I don't think we want to ship with the profiler. At the same time, enabling it for the debug builds doesn't make much sense either, since we want to profile the release. Thus, it should probably be off by default (but still be checked by CI).